src/org/parosproxy/paros/extension/encoder/Encoder.java

```diff
@@ -154,6 +155,82 @@ public String getIllegalUTF8Encode(String msg, int bytes) {
 		
 		return sbResult.toString();
 	}
+
+
+	public String getUnicodeEncode(String str) {
+        str = (str == null ? "" : str);
+        String tmp;
+        StringBuffer sb = new StringBuffer(1000);
+        char c;
+        int i, j;
+        sb.setLength(0);
+        for (i = 0; i < str.length(); i++)
+        {
+            c = str.charAt(i);
+            sb.append("%u");
+            j = (c >>>8); //取出高8位
+            tmp = Integer.toHexString(j);
+            if (tmp.length() == 1)
+                sb.append("0");
+            sb.append(tmp);
+            j = (c & 0xFF); //取出低8位
+            tmp = Integer.toHexString(j);
+            if (tmp.length() == 1)
+                sb.append("0");
+            sb.append(tmp);
+
+        }
+        return (new String(sb));
+    }
+
+    public String getUnicodeDecode(String str) {

```

Comment:
There's gotta be a better way to do this? StringUtils?

Reply:
+1 for use of a specialised library, https://github.com/unbescape/unbescape looks interesting.